src/components/account_overview/account_overview.ts

```diff
@@ -2,32 +2,40 @@ import { html, LitElement, TemplateResult } from "lit";
 import { customElement, property } from "lit/decorators.js";
 import { paymentDetailsCSS } from "./acount_overview.css";
 
+interface AccountOverviewPropObject {
+  value: number;
+  label?: string;
+  size?: "small" | "large";
+}
+
+type RenderStatParam = number | AccountOverviewPropObject | undefined;
```

Comment:
You can remove the undefined, and rely on the ?: at time of time assignment to indicate undefined state